src/SUMMARY.md

```diff
@@ -31,6 +31,7 @@
   * [PostgreSQL](configuration/services/postgresql.md)
   * [MongoDB](configuration/services/mongodb.md)
   * [InfluxDB](configuration/services/influxdb.md)
+  * [Network Storage](configuration/services/network-storage.md)
```

Comment:
Any reason why this new session be between InfluxDB and Redis? I would suggest ordering alphabetically the summary...

Reply:
The menu is largely "clustered" right now than having any specific order.  It may make sense to reorder it but that should be its own PR.